static/app/components/search/sources/apiSource.tsx

```diff
@@ -389,7 +389,10 @@ class ApiSource extends React.Component<Props, State> {
     this.handleSearchRequest(searchRequests, directRequests);
   }, 150);
 
-  handleRequestError = (err: ResponseMeta, {url, orgId}) => {
+  handleRequestError = (
+    err: ResponseMeta,
+    {url, orgId}: {orgId: Organization['id']; url: string}
```

Comment:
@JonasBa This should actually be `orgId: Organization['slug']` since the "id" is really the slug